pilot/docker/Dockerfile.proxy_debug

```diff
@@ -1,40 +1,15 @@
-FROM ubuntu:xenial
-# Do not add more stuff to this list that isn't small or critically useful.
-# If you occasionally need something on the container do
-# sudo apt-get update && apt-get whichever
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y \
-      curl \
-      iptables \
-      iproute2 \
-      iputils-ping \
-      dnsutils \
-      netcat \
-      tcpdump \
-      net-tools \
-      libc6-dbg gdb \
-      elvis-tiny \
-      lsof \
-      busybox \
-      sudo && \
-    rm -rf /var/lib/apt/lists/*
+FROM istionightly/base_debug
 
 # Install Envoy.
-ADD envoy-debug /usr/local/bin/envoy
+ADD envoy /usr/local/bin/envoy
 
 ADD pilot-agent /usr/local/bin/pilot-agent
 
 COPY envoy_pilot.json      /etc/istio/proxy/envoy_pilot.json
 COPY envoy_pilot_auth.json /etc/istio/proxy/envoy_pilot_auth.json
 COPY envoy_mixer.json      /etc/istio/proxy/envoy_mixer.json
 COPY envoy_mixer_auth.json /etc/istio/proxy/envoy_mixer_auth.json
-# Use the same file as the deb file.
 COPY envoy_bootstrap_tmpl.json /var/lib/istio/envoy/envoy_bootstrap_tmpl.json
 
-# Sudoers used to allow tcpdump and other debug utilities.
-RUN useradd -m --uid 1337 istio-proxy && \
```

Comment:
why are you removing this / where is the replacement for it ?

Reply:
In the base image.

Reply:
Ah, sorry missed the add. See docker/Dockerfile.xenial_debug